frontend/src/components/Connect/Details/Overview/Overview.styled.tsx

```diff
@@ -0,0 +1,81 @@
+import styled, { css } from 'styled-components';
+
+export const ModalOverlay = styled.div`
```

Comment:
Maybe you could create reusable component `Modal` in `components/common` ?
You are already have everything to do it and I think that there will be a lot of another cases where `kafbat/ui` will need `Modal` component.